pkg/ingester/ingester_test.go

```diff
@@ -1915,6 +1916,77 @@ func TestIngester_Push(t *testing.T) {
 				cortex_ingester_tsdb_head_max_timestamp_seconds 0.01
 			`,
 		},
+		"should not fail if native histograms are disabled": {
```

Comment:
There's a dedicated test already, but this is fine as well TestIngesterCanEnableIngestAndQueryNativeHistograms

Reply:
Nice to know - I had this one already so decided to stick with it.